openedx/adg/lms/applications/models.py

```diff
@@ -160,7 +160,7 @@ class WorkExperience(UserStartAndEndDates):
     name_of_organization = models.CharField(verbose_name=_('Name of Organization'), max_length=255, )
     job_position_title = models.CharField(verbose_name=_('Job Position / Title'), max_length=255, )
     is_current_position = models.BooleanField(verbose_name=_('Current Position'), default=False, )
-    job_responsibilities = models.TextField(verbose_name=_('Job Responsibilities'), blank=True, )
+    job_responsibilities = models.TextField(verbose_name=_('Job Responsibilities'), max_length=255, )
```

Comment:
Is it a client requirement to restrict it to 255 ? As it is a TextField and I think job responsibilities can have more text

Reply:
What's your suggestion regarding max length or you are suggesting no limit at all?